tools/db_bench_tool.cc

```diff
@@ -2363,6 +2420,15 @@ class CombinedStats {
     }
   }
 
+  double CalcStdDev(std::vector<double>& data, double average) {
+    double squared_sum = 0.0;
+    for (double x : data) {
+      squared_sum += std::pow(x - average, 2);
+    }
+
+    return std::sqrt(squared_sum / data.size() - 1);
```

Comment:
Do we get "Inf" printed now with the divide by zero? Hmm that is technically undefined behavior, so we should probably make a special case for 1 sample.

And if you're pushing another update, maybe add a comment for why the `- 1`

Reply:
Exactly, I missed that at first. Fixing the code now. I will skip printing aggregate stats when number of samples `<= 1`.